libvirt/tests/src/migration/migration_with_stress/network_load_on_host.py

```
@@ -0,0 +1,87 @@
+        Setup steps
+
+        """
+        client_ip = params.get("client_ip")
+
+        test.log.info("Setup steps.")
+        if not utils_package.package_install('netperf'):
+            test.error('Failed to install netperf on source host.')
+        if not utils_package.package_install('netperf', remote_session):
+            test.error('Failed to install netperf on target host.')
+
+        process.run("systemctl stop firewalld", shell=True, verbose=True)
```

Ruff (0.14.8)

Function call with `shell=True` parameter identified, security issue

(S604)